src/lib/itinerary.ts

```diff
@@ -34,9 +34,7 @@ interface MyResult {
   itineraries: Destination[];
 }
 
-const getLocation = () => {
-  return 'us-central1';
-};
+const aiStudioAPIKey = "YOUR_KEY_HERE";
```

Comment:
Can we switch this to use an env variable?

Reply:
This this is resolved now